api/applications.js

```diff
@@ -83,5 +84,152 @@ module.exports = [
                 }
             }
         }  
+    },
+    {
+        method: 'POST',
+        path: '/applications/{applicationId}/userSignup',
+        handler: async (request, h) => {
+            const uow = await request.app.getNewUoW();
+            const logger = request.server.app.logger;
+            const payload = request.payload;
+            let errors = [];
+
+            //send a confirmation email by default
+            let sendConfirmationEmail = true;
+            if (payload.sendConfirmationEmail != null) {
```

Comment:
This will still send the confirmation email if they set sendConfirmationEmail to be false.

Just check `if (payload.sendConfirmationEmail)`